gemfiles/rails_5.2.gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-gem "rails", "~> 5.2.0"
+gem "rails", "~> 5.2.4"
```

Comment:
this has no changes on CI, because `Gemfile.lock` isn't commited. 
it always pull the last version

it does not hurt though, just an extra commit for no reason